### ISSUE TITLE
[accessibility]: Tag Image should have alt prop 

### ISSUE THREAD

**Issue by LucasDibz** [skynexui/components#69]
## Why
- Image elements must have an alt prop, either with meaningful text, or an empty string for decorative images.

## How
- Add a `alt` prop to `<Image />` component
